src/helpers/extensionStatus.js

```diff
@@ -19,7 +19,7 @@ export const useExtensionStatus = () => {
 
     const checkExtensionStatus = () => {
       const event = document.createEvent('Event')
-      event.initEvent('checkextensionstatus')
+      event.initEvent('check-cozy-extension-status')
```

Comment:
What do you think of something more like:

cozy.passwordextension.check-status
cozy.passwordextension.installed
cozy.passwordextension.connected

Reply:
Oops I merged before seeing your comment. OK, I'm opening another PR with this change.

Reply:
See https://github.com/cozy/cozy-passwords/pull/17